Formula/o/ocicl.rb

```diff
@@ -0,0 +1,46 @@
+class Ocicl < Formula
+  desc "OCI-based ASDF system distribution and management tool for Common Lisp"
+  homepage "https://github.com/ocicl/ocicl"
+  url "https://github.com/ocicl/ocicl/archive/refs/tags/v2.3.4.tar.gz"
+  sha256 "be31c1d5de3352bf2ddc759153abc9c04ce4c0b962cc1d9d07888379e18ff202"
+  license "MIT"
+
+  depends_on "sbcl" => [:build, :test]
+  depends_on "zstd"
+  depends_on "oras"
+
+  def install
+    system "sbcl", "--dynamic-space-size", "3072", "--no-userinit", "--eval",
+           "(require 'asdf)", "--eval",
+           "(progn (push (uiop:getcwd) asdf:*central-registry*) (asdf:make :ocicl) (sb-ext:quit))"
+
+    # Write a shell script to wrap oras
+    (bin+"ocicl-oras").write <<~EOS
+      #!/bin/sh
+      oras "$@"
+    EOS
+
+    # Install the main executable
+    bin.install "ocicl"
+
+    # Make the script executable
+    chmod "+x", (bin+"ocicl-oras")
+
+    system "ls", "-l"
+    system "find", prefix
```

Comment:
```suggestion
```

Reply:
I'm trying to figure out why ocicl is failing.  When I build and install it locally, it works fine.   But when I commit my changes, I always get: 
 
 /home/linuxbrew/.linuxbrew/Cellar/ocicl/2.3.4/bin/ocicl install chat
  fatal error encountered in SBCL pid 9748 tid 9748:
  Can't find sbcl.core
  Error: ocicl: failed

This error suggests that the ocicl binary is getting stripped at install time.  We can't strip this binary, as it removes the sbcl.core file appended to the end of the executable.  Any ideas?

Reply:
I don't think we strip the binary on purpose, but it may be getting stripped by `patchelf.rb` when we try to relocate rpaths.

Reply:
Can I skip the patchelf.rb step?

Reply:
I don't think so. That said, now that I think about it, I don't think that's the cause of your problem if you're seeing it from a local `brew install --build-from-source`.

Reply:
brew install --build-from-source is working fine locally.

I'm trying a new approach where I dump the sbcl core file, and reference it from a bash script instead of binding the core to the sbcl executable.